{
	"h-How_to_authenticate_to_Wikimedia_Commons_Query_Service_using_OAuth_in_Python?-2020-12-21T13:38:00.000Z": "<span id=\"How_to_authenticate_to_Wikimedia_Commons_Query_Service_using_OAuth_in_Python.3F\" typeof=\"mw:FallbackId\"></span>How to authenticate to Wikimedia Commons Query Service using OAuth in Python?",
	"c-FrankieRayRobertson-2020-12-21T13:38:00.000Z-How_to_authenticate_to_Wikimedia_Commons_Query_Service_using_OAuth_in_Python?": "This page says it's possible to authenticate against the query service using OAuth, but has anyone managed to do so programmatically from Python? If anyone has any working code, in any language, I would appreciate whatever you can share. I've asked the full question including my not-working Python code on StackOverflow: <a rel=\"mw:ExtLink\" href=\"https://stackoverflow.com/questions/65303450/how-to-authenticate-to-wikimedia-commons-query-service-using-oauth-in-python\" class=\"external free\" id=\"mwBA\">https://stackoverflow.com/questions/65303450/how-to-authenticate-to-wikimedia-commons-query-service-using-oauth-in-python</a>",
	"c-Zache-2021-01-04T06:55:00.000Z-FrankieRayRobertson-2020-12-21T13:38:00.000Z": "<span class=\"template-ping\" about=\"#mwt2\" typeof=\"mw:Transclusion\" data-mw='{\"parts\":[{\"template\":{\"target\":{\"wt\":\"ping\",\"href\":\"./Template:Ping\"},\"params\":{\"1\":{\"wt\":\"Multichill\"}},\"i\":0}}]}' id=\"mwCw\">@<a rel=\"mw:WikiLink\" href=\"./User:Multichill\" title=\"User:Multichill\">Multichill</a>: </span> do you have example code for this / are you using WCQS with python?",
	"c-Multichill-2021-02-13T18:31:00.000Z-Zache-2021-01-04T06:55:00.000Z": "<span class=\"template-ping\" about=\"#mwt4\" typeof=\"mw:Transclusion\" data-mw='{\"parts\":[{\"template\":{\"target\":{\"wt\":\"Ping\",\"href\":\"./Template:Ping\"},\"params\":{\"1\":{\"wt\":\"Zache\"}},\"i\":0}}]}' id=\"mwEg\">@<a rel=\"mw:WikiLink\" href=\"./User:Zache\" title=\"User:Zache\">Zache</a>: </span> no, would have to look into this. Should be part of Pywikibot if someone figures it out. Did you already file a task in phabricator to add support for <a rel=\"mw:ExtLink\" href=\"https://wcqs-beta.wmflabs.org/\" class=\"external free\" id=\"mwEw\">https://wcqs-beta.wmflabs.org/</a><span typeof=\"mw:DisplaySpace\" id=\"mwFA\">Â </span>?",
	"h-HTTP_500,_server_down?-2021-12-28T00:30:00.000Z": "<span id=\"HTTP_500.2C_server_down.3F\" typeof=\"mw:FallbackId\"></span>HTTP 500, server down?",
	"c-Marsupium-2021-12-28T00:30:00.000Z-HTTP_500,_server_down?": "<p id=\"mwGA\">For me <code id=\"mwGQ\"><a rel=\"mw:ExtLink\" href=\"https://wcqs-beta.wmflabs.org/\" class=\"external free\" id=\"mwGg\">https://wcqs-beta.wmflabs.org/</a></code> throws a 500 Internal Server Error:</p>\n<pre id=\"mwGw\">Problem accessing /oauth/check_login. Reason:\nRequest failed.</pre>\n<p id=\"mwHA\">Is this known? Any information when the query service might be back? Thanks in advance for any information or linking to relevant other places!</p>",
	"c-Shisma-2021-12-31T11:48:00.000Z-Marsupium-2021-12-28T00:30:00.000Z": "<span class=\"template-ping\" about=\"#mwt6\" typeof=\"mw:Transclusion\" data-mw='{\"parts\":[{\"template\":{\"target\":{\"wt\":\"re\",\"href\":\"./Template:Re\"},\"params\":{\"1\":{\"wt\":\"Marsupium\"}},\"i\":0}}]}' id=\"mwIw\">@<a rel=\"mw:WikiLink\" href=\"./User:Marsupium\" title=\"User:Marsupium\">Marsupium</a>: </span> issue was reported in T297454",
	"c-Marsupium-2021-12-31T12:19:00.000Z-Shisma-2021-12-31T11:48:00.000Z": "Thanks for the link!"
}
